mne/gui/_viewer.py

```diff
@@ -291,7 +284,7 @@ def clear(self):  # noqa: D102
     @on_trait_change('scene.activated')
     def plot(self):
         """Add the points to the mayavi pipeline"""
-        _scale = self.scene.camera.parallel_scale if not _testing_mode() else 1
+        _scale = self.scene.camera.parallel_scale
```

Comment:
Hm, the GUI tests work on OS X but not on linux. Is it possible there is something about the environment? Mayavi's [travis.yml](https://github.com/enthought/mayavi/blob/master/.travis.yml) starts `/etc/init.d/xvfb` [here](https://github.com/enthought/mayavi/blob/master/travis/before_install.sh), but our `travis.yml` also seems to run it, just in a different way?